server/app/services/applicant/question/NameQuestion.java

```diff
@@ -102,6 +103,20 @@ public Optional<String> getLastNameValue() {
     return lastNameValue;
   }
 
+  public Optional<String> getSuffixValue() {
+    if (suffixValue != null) {
+      return suffixValue;
+    }
+
+    ApplicantData applicantData = applicantQuestion.getApplicantData();
+    suffixValue = applicantData.readString(getSuffixPath());
+
+    if (middleNameValue.isEmpty() && isPaiQuestion()) {
+      middleNameValue = applicantData.getApplicantNameSuffix();
```

Comment:
**issue**: Should this be checking and setting the `middleNameValue`, or is it a bug?